api/spec/serializers/spree/v2/storefront/cms_section_serializer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Spree::V2::Storefront::CmsSectionSerializer do
```

Comment:
@tomurb please add different tests for different section types - https://github.com/spree/spree/tree/master/core/app/models/spree/cms/sections